### PR TITLE
Updates contribution scores info and screenshots in risk scoring docs

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Contributes to https://github.com/elastic/security-docs/issues/5046.

Previews:
* [Hosts page](https://security-docs_bk_5071.docs-preview.app.elstc.co/guide/en/security/master/hosts-overview.html)
* [Users page](https://security-docs_bk_5071.docs-preview.app.elstc.co/guide/en/security/master/users-page.html)

Twin serverless PR: https://github.com/elastic/staging-serverless-security-docs/pull/323

### CODE REVIEW

File: docs/getting-started/users-page.asciidoc [hunk 1]
```diff
@@ -85,7 +85,12 @@ The risk summary visualization shows the user risk score and user risk level. Ho
 
 The risk summary table shows the category, score, and number of risk inputs that determine the user risk score. Hover over the table to display the **Inspect** button, which allows you to inspect the table's queries.
 
-To expand the **User risk summary** section, click **View risk contributions**. The left panel displays additional details about the user's risk inputs.
+To expand the **User risk summary** section, click **View risk contributions**. The left panel displays additional details about the user's risk inputs:
+
+* The asset criticality level and contribution score from the latest risk scoring calculation.
+* The top 10 alerts that contributed to the latest risk scoring calculation, and each alert's contribution score.
```
Comment (benironside) [inline docs/getting-started/users-page.asciidoc:91]:
```suggestion
* The top 10 alerts that contributed to the latest risk score, and each alert's contribution score.
```

File: docs/getting-started/users-page.asciidoc [hunk 2]
```diff
@@ -85,7 +85,12 @@ The risk summary visualization shows the user risk score and user risk level. Ho
...
+
+If more than 10 alerts contributed to the risk scoring calculation, the remaining alerts' aggregated contribution score is displayed below the **Alerts** table.
```
Comment (benironside) [inline docs/getting-started/users-page.asciidoc:93]:
```suggestion
If more than 10 alerts contributed to the risk score, the remaining alerts' aggregate contribution score is displayed below the **Alerts** table.
```

File: docs/management/hosts/hosts-overview.asciidoc [hunk 3]
```diff
@@ -88,7 +88,12 @@ The risk summary visualization shows the host risk score and host risk level. Ho
 
 The risk summary table shows the category, score, and number of risk inputs that determine the host risk score. Hover over the table to display the **Inspect** button, which allows you to inspect the table's queries.
 
-To expand the **Host risk summary** section, click **View risk contributions**. The left panel displays additional details about the host's risk inputs.
+To expand the **Host risk summary** section, click **View risk contributions**. The left panel displays additional details about the host's risk inputs:
+
+* The asset criticality level and contribution score from the latest risk scoring calculation.
+* The top 10 alerts that contributed to the latest risk scoring calculation, and each alert's contribution score.
+
+If more than 10 alerts contributed to the risk scoring calculation, the remaining alerts' aggregated contribution score is displayed below the **Alerts** table.
```
Comment (benironside) [inline docs/management/hosts/hosts-overview.asciidoc:96]:
```suggestion
If more than 10 alerts contributed to the risk score, the remaining alerts' aggregate contribution score is displayed below the **Alerts** table.
```

File: docs/management/hosts/hosts-overview.asciidoc [hunk 4]
```diff
@@ -88,7 +88,12 @@ The risk summary visualization shows the host risk score and host risk level. Ho
 
 The risk summary table shows the category, score, and number of risk inputs that determine the host risk score. Hover over the table to display the **Inspect** button, which allows you to inspect the table's queries.
 
-To expand the **Host risk summary** section, click **View risk contributions**. The left panel displays additional details about the host's risk inputs.
+To expand the **Host risk summary** section, click **View risk contributions**. The left panel displays additional details about the host's risk inputs:
+
+* The asset criticality level and contribution score from the latest risk scoring calculation.
+* The top 10 alerts that contributed to the latest risk scoring calculation, and each alert's contribution score.
```
Comment (benironside) [inline docs/management/hosts/hosts-overview.asciidoc:94]:
```suggestion
* The top 10 alerts that contributed to the latest risk score, and each alert's contribution score.
```

**Review by benironside — APPROVED** [review]
LGTM - left a couple optional suggestions

File: docs/getting-started/users-page.asciidoc (hunk 1 above)
Comment (natasha-moore-elastic) [reply docs/getting-started/users-page.asciidoc:91]: I feel that "risk scoring calculation" is a bit more precise here, so I'd prefer to leave it as is.

**Review by benironside — APPROVED** [review]

**Pull request merged**